server/xpub-controller/index.js

```diff
@@ -1,9 +1,11 @@
 const SupportingFiles = require('./supportingFiles')
 const Manuscript = require('./manuscript')
 const S3Storage = require('./s3Storage')
+const ScienceBeamApi = require('./scienceBeamApi')
```

Comment:
Would be good to create a `xpub-client` package to contain the `ScienceBeamApi`, and the `S3Storage`, since it's an isolated piece of functionality.

Reply:
It also means we can rework the controller to follow any pattern in the future, including redux or something.